apps/condo/lang/ru/messages/TRACK_TICKET_IN_DOMA_APP/default.njk

```diff
@@ -0,0 +1 @@
+Следите за вашей заявкой в приложении Doma: http://onelink.to/doma
```

Comment:
It’s better to remove hard coded links, but probably no in this pr. It looks like another task